CODEOFCONDUCT.md

```diff
@@ -1,27 +1,28 @@
 # Summer of Hacks Code of Conduct
 
-Our events are dedicated to providing a harassment-free event experience for everyone, regardless of gender, age, sexual orientation, disability, physical appearance, body size, race, or religion (or lack thereof). We do not tolerate harassment of event participants in any form. Sexual language and imagery is not appropriate for any event venue. Event participants violating these rules may be sanctioned or expelled from the event without a refund (if applicable) at the discretion of the event organisers.
+Our events are dedicated to providing a harassment-free event experience for everyone, regardless of gender, age, sexual orientation, disability, physical appearance, body size, race, or religion (or lack thereof). We do not tolerate harassment of event participants in any form. Sexual language and imagery is not appropriate for any event venue. Event participants violating these rules may be sanctioned or expelled from the event at the discretion of the event organisers.
 
 *tl;dr: Be excellent with each other*
 
 ## Need Help?
 
-The JSOxford organisers are there to help you have a good time. If you have any questions, concerns or need help, please reach out to us:
+The Summer of Hacks organisers are there to help you have a good time. If you have any questions, concerns or need help, please reach out to us:
 
-- Ryan Brooks: 07841 757984, ryan@ryanbrooks.co.uk (slack - @ryan)
-- Ben Foxall: 07540838982, benfoxall@gmail.com (slack - @ben)
-- Pete West: 07565631681, peterjwest3@gmail.com (slack - @peterjwest)
+- Adam Leskis (slack - @aleskis)
```

Comment:
Do add whatever deets you like and deem public enough: phone, Twitter handle, etc.

> If Twitter, then consider making your DMs open around the time of events.